src/main/java/duke/tasklist/TaskList.java

```diff
@@ -0,0 +1,144 @@
+package duke.tasklist;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import duke.duke_exception.DukeException;
```

Comment:
Consider leaving a blank line separator at line 8 to follow the Java Coding Standards outlined for package imports to separate "java" imports and "duke" imports.

Reply:
Sounds good.